### PR TITLE
chore: add language field to Recipient

### PR DESCRIPTION
**Pull request by ono-max**
Part of https://github.com/bucketeer-io/bucketeer/issues/325

Recipient is one of the columns in the subscription field. Here is the process flow.

1. Assign subscription.Recipient.Language in [send method](https://github.com/bucketeer-io/bucketeer/blob/main/pkg/notification/sender/sender.go#L114)
2. Create localizer in pkg/notification/sender/notifier/slack.go.
3. Call `locale.NewLocalizer`.

### CODE REVIEW

File: proto/notification/recipient.proto [hunk 1]
```diff
@@ -19,8 +19,13 @@ option go_package = "github.com/bucketeer-io/bucketeer/proto/notification";
 
 message Recipient {
   enum Type { SlackChannel = 0; }
+  enum Language {
+    Japanese = 0;
```
Comment (cre8ivejp) [inline proto/notification/recipient.proto:23]: Can you set English as the first option?

Comment (cre8ivejp) [reply proto/notification/recipient.proto:23]: Also, please use upper case letters for enum values.

**Review by cre8ivejp — APPROVED** [review]
👍

**Review by masaaania — APPROVED** [review]
LGTM!

**Pull request merged**